charts/platform-service/templates/deployment.yaml

```diff
@@ -135,6 +133,18 @@ spec:
               value: {{ $value | quote }}
 {{- end }}
 {{- end }}
+{{- if eq .Values.keyVaultSecrets.enabled true }}

```

Comment:
now that we have a separate chart for importing secrets from keyvault I think this should be removed as we can just use regular syntax for getting env vars from secrets.

Reply:
To be considered